Mulligan.1.html

```diff
@@ -0,0 +1,174 @@
+<html>
+    <head>
+            <style>
+                 div.padded {  
```

Comment:
Try not to create generic selectors like `.padded`. Down the road, you might find yourself using variations of padding.

```
.container-something {
     padding: 50px 400px 400px;
}
```

Use shorthand whenever possible.

` padding: 50px 400px 50px 400px;`

Read: https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties